superset-frontend/src/explore/controlPanels/Compare.js

```diff
@@ -19,6 +19,18 @@
 import { t } from '@superset-ui/translation';
 import { NVD3TimeSeries, annotations } from './sections';
 import { D3_TIME_FORMAT_OPTIONS } from '../controls';
+import {
+  xAxisLabel,
+  yAxisLabel,
+  bottomMargin,
+  xTicksLayout,
+  xAxisFormat,
+  yLogScale,
+  yAxisBounds,
+  xAxisShowMinmax,
+  yAxisShowMinmax,
+  leftMargin,
+} from './Shared_NVD3';
```

Comment:
Perhaps use camelcase file naming `SharedNVD3`.

Reply:
Hmm... I was just going for namespacing, even if temporary. RIght now there are:
`Shared_NVD3.js`
`Shared_DeckGL.jsx`
`Shared_BigNumber.js`

I imagine when we pull all this stuff into the individual packages, we can just call them all `SharedControls.js(x)`

If you'd like to get rid of the underscores in the interim (no strong feelings, here), I might rather just fix all three in a fast-follow PR after this one.

Reply:
Thanks for the clarification.